<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">background-color: #2f3e40; /*#2f3e46,1a535c;1d3557;233d4d*/
color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #2f3e40; /*#2f3e46,1a535c;1d3557;233d4d*/
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Bon de visite</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>510</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #52796f;</string>
           </property>
           <property name="text">
            <string>NÂ° Dossier</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="rechercher_bon_visite">
           <property name="styleSheet">
            <string notr="true">border : 1px solid #2f3e46;
border-radius:10px;
padding: 3px;
background-color: white;
color: #2f3e46;
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QTableWidget" name="bon_visite_table">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: white;
color: #52796f;
selection-background-color: #a2d2ff;
selection-color:#2f3e46;
border-radius: 10px;
</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>declarant</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>num_dossier</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>agent</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>transit</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="nouveau_bon_visite">
             <property name="styleSheet">
              <string notr="true">QPushButton {width: 65px;
background-color: white;
border: 2px solid #3bb273;
color: #3bb273;
padding: 6px;
border-radius: 10px;}

QPushButton:hover {background-color: #3bb273;color: white;}
QPushButton:pressed { margin: 1px; }
</string>
             </property>
             <property name="text">
              <string>Nouveau</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="modifier_bon_visite">
             <property name="styleSheet">
              <string notr="true">QPushButton {width: 65px;
background-color: white;
border: 2px solid #3bb273;
color: #3bb273;
padding: 6px;
border-radius: 10px;}

QPushButton:hover {background-color: #3bb273;color: white;}
QPushButton:pressed { margin: 1px; }
</string>
             </property>
             <property name="text">
              <string>Modifier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="supprimer_bon_visite">
             <property name="styleSheet">
              <string notr="true">QPushButton {background-color: white;
border: 2px solid #ee2e31;
color: #ee2e31;
padding: 5px;
border-radius: 10px;}
QPushButton:hover {
background-color: #ee2e31;
color: white;
}
QPushButton:pressed{
margin: 1px;
}
/*#ee2e31*/</string>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fermer_bon_visite">
             <property name="styleSheet">
              <string notr="true">QPushButton {background-color: white;
border: 2px solid #ee2e31;
color: #ee2e31;
padding: 5px;
border-radius: 10px;}
QPushButton:hover {
background-color: #ee2e31;
color: white;
}
QPushButton:pressed{
margin: 1px;
}
/*#ee2e31*/</string>
             </property>
             <property name="text">
              <string>Fermer</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
